pyiceberg/table/update/snapshot.py

```diff
@@ -239,7 +240,21 @@ def _summary(self, snapshot_properties: Dict[str, str] = EMPTY_DICT) -> Summary:
             truncate_full_table=self._operation == Operation.OVERWRITE,
         )
 
+    def refresh(self) -> TableMetadata:
+        try:
+            table = self._transaction._table.refresh()
+            return table.metadata
+        except Exception:
+            return self._transaction._table.metadata
+
+    @abstractmethod
+    def _validate(self, current_metadata: TableMetadata, Snapshot: Optional[Snapshot]) -> None: ...
```

Comment:
Variable names should be lower-case:
```suggestion
    def _validate(self, current_metadata: TableMetadata, snapshot: Optional[Snapshot]) -> None: ...
```

Reply:
fixed